Makefile

```diff
@@ -18,8 +18,9 @@ TESTS = $(shell find test -name "*.js" -type f | sort)
 
 all: mocha.js
 
-mocha.js: $(SRC) browser-entry.js
+mocha.js BUILDTMP/mocha.js: $(SRC) browser-entry.js
```

Comment:
I think it's worth noting that this makes the directory quite obvious (not only is it not a dot-directory like `.buildtmp`, but it's in CAPs); I assume this is intentional?